pageserver/src/layered_repository/inmemory_layer.rs

```diff
@@ -54,6 +60,33 @@ pub struct InMemoryLayerInner {
 }
 
 impl Layer for InMemoryLayer {
+
+    // An in-memory layer doesn't really have a filename as it's not stored on disk,
+    // but we construct a filename as if it was a delta layer
+    fn filename(&self) -> PathBuf {
```

Comment:
Do we use it anywhere or it's just for debugging?

Reply:
Only in some log messages in `LayeredTimeline::load_layer_map`